### ISSUE TITLE
Convert Existing team

### ISSUE THREAD

**Issue by jarvisington** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#1054]
Is it possible to convert an existing team to an Org-Wide team?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 76309c0b-2a9c-36a8-d7c7-f7d6f311d39b
* Version Independent ID: bc69c0e3-0fa7-c86e-5e8c-067c8811e9bf
* Content: [Create an org-wide team in Microsoft Teams](https://docs.microsoft.com/en-us/MicrosoftTeams/create-an-org-wide-team)
* Content Source: [Teams/create-an-org-wide-team.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/create-an-org-wide-team.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by rico-neuendorf** [comment]
also interessted!

**Comment by PhilFancy** [comment]
interested as well!

**Comment by cjmain** [comment]
yes please

**Comment by expiscornovus** [comment]
Agree, nice idea :)

**Comment by aThuemmel** [comment]
As it adds just everyone from you Azure AD, exisiting users will still be member, just all other users will be added to the Office Group behind the Team. Files, Posts etc. dont change. It is the same as you would just add all these members manually.

**Comment by sdecker** [comment]
No, it is not possible to change an existing team to Org-Wide. I've tried and it doesn't work. I even opened a support case and gone round and round with support and they have unequivocally stated it cannot be done. The paragraph in the FAQ that says you can is a lie.

**Comment by nswetland** [comment]
Yes.  I just converted a team today from public to Org-Wide.   Originally it did not let me, and kept saying I was unauthorized (I was a Global Admin of O365 and Owner on the Team).   The only change I made recently that might have helped was switching my MS Teams coexistance mode to "Teams Only" (it was Islands, but we killed SfB in Q4 2018).   Not sure if that is what fixed it, but it definitely let me do it.
